snowshu/configs.py

```diff
@@ -34,6 +36,17 @@ def _is_in_docker() -> bool:
         return False
 
 
+def _get_architecture() -> str:
+    """
+     Returns the machine type. An empty string is returned if the value cannot be determined.
+    """
+    iso_arch = platform.machine()
```

Comment:
What does the Apple M1 return for this value?

Some quick testing on my side ended up with both in and out of a docker image returning `x86_64`

Reply:
1) local run:
![image](https://user-images.githubusercontent.com/98582037/175508441-843bc9d9-c9db-402f-8609-480568e39fd2.png)
2) docker run(the arm image):
![image](https://user-images.githubusercontent.com/98582037/175508524-f55fb20f-6898-42b4-8690-72ccb811cf96.png)

Reply:
we are getting the following warning(in case of arch mismatch):
![image](https://user-images.githubusercontent.com/98582037/175517064-8d502ad9-095c-4db3-829b-880dd6c21f20.png)